### PR TITLE
fix: remove linefeeds at the end

### PR DESCRIPTION
**Pull request by dominique-pfister**

### CODE REVIEW

File: src/coralogix.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ export class CoralogixLogger {
             invocationId: extractedFields.request_id || 'n/a',
             functionName: this._funcName,
           },
-          message,
+          message: message.replace(/\n+$/, ''),
```
Comment (dominique-pfister) [inline src/coralogix.js:52]: remove linefeeds at the end, only

Comment (tripodsan) [reply src/coralogix.js:52]: why not all ws? `message.trimEnd()` ?

Comment (dominique-pfister) [reply src/coralogix.js:52]: there is `trimEnd`? Every day is a school day 😊

**Pull request merged**